### DISCUSSION TITLE
How to login into Microcks via Keycloak

### DISCUSSION THREAD

**Discussion by CROprogrammer** [microcks/microcks#839] [Q&A] [answered]
Hi,
I've deployed Microks on my local Openshift using Microcks operator and MicrocksInstall resource. 
When I go to exposed route to my Microcks app, then first I have to login via Keycloak.
I try to login with username and password from `microcksinstall-keycloak-admin` secret but it says that those credientials are invalid.
What am I doing wrong?

**Comment by CROprogrammer** [comment, marked as answer]
Nevermind, this video explains it: https://www.youtube.com/watch?v=P2RFDxL0xsA&ab_channel=Microcks

**Reply by lbroudoux** [reply]
Awesome! Thanks for having taken the time to put the reference here.
